src/main/java/org/craftercms/studio/api/v1/constant/StudioConstants.java

```diff
@@ -148,7 +150,14 @@
 
     public static final String INDEX_FILE = "index.xml";
 
-    public static final String HOME_PAGE_PATH = "/site/website/index.xml";
+    public static final List<String> TOP_LEVEL_FOLDERS = List.of(
+        "/site/website/index.xml",
+        "/site/components",
+        "/site/taxonomy",
+        "/static-assets",
+        "/templates",
+        "/scripts"
+    );
```

Comment:
In support of the headless BPs and the various VCs, should this list also include /items, /videos, /streams (was it streams?).

Also, we may be standardizing on /source (verify with Roy). So we may want to include that as well.